### PR TITLE
refactor: Re-rule the RST files in this repo.

### PR DESCRIPTION
**Pull request by feanil**
**Description:**

Re-rule the titles so that they follow the guidance provided by
docs.openedx.org

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

### CODE REVIEW

**Comment by feanil** [comment]
Current dependencies on/for this PR:
* master
  * **PR #240** <a href="https://app.graphite.dev/github/pr/openedx/edx-cookiecutters/240" target="_blank">Graphite</a>  👈
    * **PR #241** <a href="https://app.graphite.dev/github/pr/openedx/edx-cookiecutters/241" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/openedx/edx-cookiecutters/240?utm_source=stack-comment).

File: python-template/{{cookiecutter.placeholder_repo_name}}/CHANGELOG.rst [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+**********
 
 *
 
 [{{ cookiecutter.version }}] - {% now 'local' %}
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+************************************************
 
 Added
 _____
```
Comment (robrap) [inline python-template/{{cookiecutter.placeholder_repo_name}}/CHANGELOG.rst:23]: I think this needs to be updated.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,45 +1,53 @@
 Change Log
-==========
+##########
```
Comment (robrap) [inline CHANGELOG.rst:2]: I noticed ">>>>>>> master" in this file when reviewing. Would you mind cleaning that up?

**Review by robrap — COMMENTED** [review]
Minor comments. Thanks.

**Review by robrap — APPROVED** [review]

**Pull request merged**